pkgs/development/python-modules/scikit-image/default.nix

```diff
@@ -43,12 +52,41 @@ buildPythonPackage rec {
 
   checkInputs = [ pytest ];
 
-  # No tests in archive
-  doCheck = false;
+  # See the explanation above
+  checkPhase = ''
+    rm -r skimage
+    export INSTALL_PATH=$out/${python.sitePackages}
```

Comment:
Why is this necessary?

Reply:
`INSTALL_PATH` is just a helper variable to form paths to test modules in the nix store. It points to the installed package path.